parsel.ts

```diff
@@ -431,6 +503,10 @@ export interface Tokens {
 	argument?: string;
 	subtree?: AST;
 	caseSensitive?: 'i';
+	/**
+	 * @internal
```

Comment:
We still want to document what this is (which was non-obvious to me)

Reply:
Fixed. (No longer relevant)